.github/workflows/ui-tests.yaml

```diff
@@ -0,0 +1,101 @@
+name: UI Tests
+
+on:
+  push:
+    branches:
+      - master
+      - dev-*
+      - release-*
+  pull_request:
+    branches:
+      - master
+      - dev-*
+      - release-*
+
+env:
+  DEVELOPER_DIR: /Applications/Xcode_13.3.app/Contents/Developer
```

Comment:
will this break for future releases? also couldn't find where it's being used. Is it needed?

Reply:
When you set that environment variable, `xcodebuild` will automatically stitch to the version at that path temporarily. 

And in the future I plan to run against `latest` once GitHub Actions switches it's default.